eng/pipelines/handlers.yml

```diff
@@ -120,4 +120,111 @@ stages:
           provisionator.path : '$(System.DefaultWorkingDirectory)/eng/provisioning/provisioning.csx'
           provisionator.extraArguments : '--v'
         steps:
-          - template: common/controlgallery-android.yml
\ No newline at end of file
+          - template: common/controlgallery-android.yml
+  
+  - stage: build_net6_windows
```

Comment:
These YAML steps were taken from here

https://github.com/xamarin/net6-samples/blob/main/azure-pipelines.yml

For installing the latest NET6 bits and such
At some point this will all get moved into easier to provision bits via the dotnet tool